src/main/java/com/exasol/adapter/dynamodb/mapping/AbstractColumnMapping.java

```diff
@@ -20,4 +20,21 @@ protected AbstractColumnMapping(final String exasolColumnName) {
     public final String getExasolColumnName() {
         return this.exasolColumnName;
     }
+
```

Comment:
Update serial UID.

Reply:
I changed ot for all the clases, but Im think it would not have been needed, as i only changed the implementation of quals and not the fields. Therefore serialization should still work?

Reply:
Yes, you are right, that is correct.